### PR TITLE
modify quotaAndLimits params

### PR DESCRIPTION
**Pull request by Bennu-Li**
Signed-off-by: Bennu-Li <yunmei.li@zilliz.com>

## Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[mychartname]`)
- [ ] PR only contains changes for one chart

### CODE REVIEW

File: charts/milvus/README.md [hunk 1]
```diff
@@ -139,30 +139,29 @@ The following table lists the configurable parameters of the Milvus Service and
 | `quotaAndLimits.ddl.eabled`               | Enable milvus ddl limit                       | `false`                                                |
 | `quotaAndLimits.ddl.collectionRate`       | Milvus ddl collection rate per minute         | `unset`                                                |
```
Comment (bigsheeper) [inline charts/milvus/README.md:140]: please modify to qps instead of *per minute* :)

File: charts/milvus/README.md [hunk 2]
```diff
@@ -139,30 +139,29 @@ The following table lists the configurable parameters of the Milvus Service and
...
 | `quotaAndLimits.ddl.partitionRate`        | Milvus ddl partition rate per minute          | `unset`                                                |
```
Comment (bigsheeper) [inline charts/milvus/README.md:141]: same as above

File: charts/milvus/README.md [hunk 3]
```diff
@@ -139,30 +139,29 @@ The following table lists the configurable parameters of the Milvus Service and
...
-| `quotaAndLimits.ddl.indexRate`            | Milvus ddl index rate per minute              | `unset`                                                |
-| `quotaAndLimits.ddl.flushRate`            | Milvus ddl flush rate per minute              | `unset`                                                |
-| `quotaAndLimits.ddl.compactionRate`       | Milvus ddl compaction rate per minute         | `unset`                                                |
+| `quotaAndLimits.indexRate.eabled`         | Enable milvus index rate limit                | `false`                                                |
+| `quotaAndLimits.indexRate.max`            | Milvus max index rate per minute              | `unset`                                                |
```
Comment (bigsheeper) [inline charts/milvus/README.md:143]: same as above

File: charts/milvus/README.md [hunk 4]
```diff
@@ -139,30 +139,29 @@ The following table lists the configurable parameters of the Milvus Service and
...
+| `quotaAndLimits.flushRate.eabled`         | Enable milvus flush rate limit                | `false`                                                |
+| `quotaAndLimits.flushRate.max`            | Milvus max flush rate per minute              | `unset`                                                |
```
Comment (bigsheeper) [inline charts/milvus/README.md:145]: same as above

File: charts/milvus/README.md [hunk 5]
```diff
@@ -139,30 +139,29 @@ The following table lists the configurable parameters of the Milvus Service and
...
+| `quotaAndLimits.compactionRate.eabled`    | Enable milvus compaction rate limit           | `false`                                                |
+| `quotaAndLimits.compactionRate.max`       | Milvus max compaction rate per minute         | `unset`                                                |
```
Comment (bigsheeper) [inline charts/milvus/README.md:147]: same as above

**Review by LoveEachDay — APPROVED** [review]
/lgtm

**Pull request merged**